tests/shadow/test_shadow_measurement.py

```diff
@@ -160,14 +160,23 @@ def test_measurement_process_copy(self, wires, shots, seed):
     def test_format(self, wires, shots, seed, interface, device):
         """Test that the format of the returned classical shadow
         measurement is correct"""
+        import tensorflow as tf
         import torch
 
         circuit = get_circuit(wires, shots, seed, interface, device)
         shadow = circuit()
 
-        # test shape and dtype are correct
+        # test shape is correct
         assert shadow.shape == (2, shots, wires)
-        assert shadow.dtype == np.uint8 if interface != "torch" else torch.uint8
```

Comment:
is this a mistake from merging or intentional?

Reply:
It's intentional. The dtype test is immediately below.